### PR TITLE
issue(#729) Fix dropdown select when used in web component

### PR DESCRIPTION
**Pull request by kubiq**
issue (#729)

### CODE REVIEW

File: src/ng-select/ng-dropdown-panel.component.ts [hunk 1]
```diff
@@ -207,6 +207,14 @@ export class NgDropdownPanelComponent implements OnInit, OnChanges, OnDestroy, A
             return;
         }
 
+        if (
+            $event.target
```
Comment (varnastadeus) [inline src/ng-select/ng-dropdown-panel.component.ts]: add unit tests

Comment (kubiq) [reply src/ng-select/ng-dropdown-panel.component.ts]: done

**Comment by benoitjchevalier** [comment]
We're also waiting on this PR to be merged, thanks for your work :)

**Comment by nikkonduru88** [comment]
Thanks for the fix! This issue is fixed in Chrome, but not sure why still seeing this issue in Safari. Anybody else seeing this?

**Pull request merged**